.github/workflows/notify_test_workflow.yml

```diff
@@ -0,0 +1,40 @@
+name: Notify test workflow
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  notify:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: "Notify test workflow"
+        uses: actions/github-script@v3
+        if: ${{ github.base_ref == 'master' }}
```

Comment:
What happens for the backport PR to branch-3.0/branch-3.1? Shall we just run the test in Apache/Spark?

Reply:
I haven't read the code yet, but backport PRs are not common and I think it's OK to use the ASF resource to run tests for them.

Reply:
Yeah, let's keep it simple for now.  We will need to update the file `.github/workflows/build_and_test.yml` in branch-3.2. We can try finding a way to make it work for every new release branch later. (Simply manually update is also fine)